csv/phases.csv

```diff
@@ -0,0 +1,14 @@
+Phase.SETUP
+Phase.SETUP
+Phase.SETUP
+Phase.SETUP
+Phase.TRAVERSE
+Phase.RETURN
+Phase.TRAVERSE
+Phase.RETURN
+Phase.TRAVERSE
+Phase.RETURN
+Phase.TRAVERSE
+Phase.RETURN
+Phase.TRAVERSE
+Phase.RETURN
```

Comment:
```suggestion
```
CSV data in phases.csv should be kept empty for our storage size, so delete csv data in this file.